### ISSUE TITLE
Migration fails with 'CHECK CONSTRAINT'

### ISSUE THREAD

**Issue by VaibhaveS** [dimitri/pgcopydb#498]
**Logs**

```
11:04:13 856500 INFO   cli_clone_follow.c:581    STEP 10: restore the post-data section to the target database
11:04:13 856500 NOTICE pgcmd.c:948                /usr/bin/pg_restore -f /tmp/pgcopydb/schema/post-out.list -l /tmp/pgcopydb/schema/post.dump
11:04:13 856500 ERROR  pgcmd.c:1218              Failed to parse Archive TOC: 4098; 2606 140079 CHECK CONSTRAINT public employees chk_employees_name_not_abc azureuser
11:04:13 856500 ERROR  pgcmd.c:1108              Failed to parse line 15 of "/tmp/pgcopydb/schema/post-out.list", see above for details
11:04:13 856500 ERROR  dump_restore.c:277        Failed to prepare the pg_restore --use-list catalogs, see above for details
```

**Repro steps**

```
CREATE TABLE public.employees (
    name character varying NOT NULL,
    emp_id uuid
);
 
ALTER TABLE public.employees
    ADD CONSTRAINT chk_employees_name_not_abc CHECK (((name)::text !~ 'abc'::text)) NOT VALID;
```

**Comment by lospejos** [comment]
It seems, that `pgcopydb` does not restore check constraints, created as `NOT VALID` on source database.

**Comment by VaibhaveS** [comment]
Fixed by #497
